src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,172 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    class Node<T> {
```

Comment:
inner classes should be declared at the very end of the class
```suggestion
    private static class Node<T> {
```